### ISSUE TITLE
Add output collection to MOVE.

### ISSUE THREAD

**Issue by flackr** [ArnoldSmith86/virtualtabletop#1403]
When moving cards I often want to set their position or take some other action on them. I regularly find myself having to SELECT them to do so, e.g.

```
      {
        "func": "MOVE",
        "from": "${from}",
        "to": "${to}",
        "count": 1
      },
      {
        "func": "SELECT",
        "property": "parent",
        "relation": "==",
        "value": "${to}",
        "sortBy": {
          "key": "z",
          "reverse": true
        },
        "collection": "DEFAULT"
      },
      {
        "func": "SELECT",
        "source": "DEFAULT",
        "property": "parent",
        "relation": "==",
        "value": "${to}",
        "collection": "DEFAULT",
        "max": 1
      },
      {
        "collection": "DEFAULT",
        "func": "SET",
        "property": "x",
        "relation": "=",
        "value": "${x}",
      },
      {
        "collection": "DEFAULT",
        "func": "SET",
        "property": "y",
        "relation": "=",
        "value": "${y}",
      }
```

It would be nice to be able to specify a collection to receive the moved widgets (e.g. similar to [CLONE](https://github.com/ArnoldSmith86/virtualtabletop/wiki/Functions#clone)) which would shorten the above to the following:

```
      {
        "func": "MOVE",
        "from": "${from}",
        "to": "${to}",
        "count": 1,
        "collection": "DEFAULT"
      },
      {
        "collection": "DEFAULT",
        "func": "SET",
        "property": "x",
        "relation": "=",
        "value": "${x}",
      },
      {
        "collection": "DEFAULT",
        "func": "SET",
        "property": "y",
        "relation": "=",
        "value": "${y}",
      }
```

**Comment by robartsd** [comment]
We want to enable support calling MOVE with collection providing the widgets to move rather than a holder as well, but I this can reasonably be added without finishing that.

**Comment by flackr** [comment]
Yep, I'm aware. It is interesting though that if you did that first you might reasonably use `collection` as the property to move from, but if you follow the pattern from CLONE it would be more consistent for collection to be the output collection and to use source as the source collection.

**Comment by flackr** [comment]
Silly me. They would be the same collection in this case anyways.

**Comment by flackr** [comment]
Just thinking out loud - it would be nice to have `collection` default to `DEFAULT`, but if we do this we may want to support null so that we can rewrite existing calls to MOVE to pass `collection: null` so that they don't overwrite the DEFAULT collection in cases where the game didn't expect it previously.

**Comment by robartsd** [comment]
> Yep, I'm aware. It is interesting though that if you did that first you might reasonably use `collection` as the property to move from, but if you follow the pattern from CLONE it would be more consistent for collection to be the output collection and to use source as the source collection.

Yes, for existing routine operations that output a collection, we use `collection` parameter to name the output collection and `source` to name the input collection. Existing routines that accept input collections but do not create output collections simply use `collection` as the input collection parameter. `source` is also used as the parameter name for a resource URL in the `AUDIO`. But renaming 'collection' to 'source' for `MOVE` and `MOVEXY` would have been easily done in a fileupdater routine.

**Comment by robartsd** [comment]
> Just thinking out loud - it would be nice to have `collection` default to `DEFAULT`, but if we do this we may want to support null so that we can rewrite existing calls to MOVE to pass `collection: null` so that they don't overwrite the DEFAULT collection in cases where the game didn't expect it previously.

I think I'd prefer that the default remain 'null'; I'd rather the code be explicit about the side effect of MOVE (of course similar argument could be made for CLONE - the primary function is ).
